### PR TITLE
Optimize performance, log MDC information only when sql-show is enabled.

### PR DESCRIPTION
**Pull request by RaigorJiang**
Optimize performance, log MDC information only when sql-show is enabled.

### CODE REVIEW

File: shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java [hunk 1]
```diff
@@ -66,8 +69,9 @@
     @Override
     public void run() {
         boolean isNeedFlush = false;
+        boolean sqlShowEnabled = isSQLShowEnabled();
         try (PacketPayload payload = databaseProtocolFrontendEngine.getCodecEngine().createPacketPayload((ByteBuf) message, context.channel().attr(CommonConstants.CHARSET_ATTRIBUTE_KEY).get())) {
-            fillLogMDC();
+            fillLogMDC(sqlShowEnabled);
```
Comment (TeslaCN) [inline shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java:74]: Could we consider checking the value before entering the method?

Comment (RaigorJiang) [reply shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java:74]: Sure, thanks for your advice.  😀

File: shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java [hunk 2]
```diff
@@ -66,8 +69,11 @@
     @Override
     public void run() {
         boolean isNeedFlush = false;
+        boolean sqlShowEnabled = isSQLShowEnabled();
```
Comment (TeslaCN) [inline shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java:72]:
```suggestion
        boolean sqlShowEnabled = ProxyContext.getInstance().getContextManager().getMetaDataContexts().getProps().getValue(ConfigurationPropertyKey.SQL_SHOW);
```
Could we consider doing inline manually?

Comment (RaigorJiang) [reply shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java]: Thank you @TeslaCN , updated.

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**